libvmaf/include/libvmaf/model.h

```diff
@@ -1,9 +1,11 @@
 #ifndef __VMAF_MODEL_H__
 #define __VMAF_MODEL_H__
 
+#include <libvmaf/model.config.h>
+
 typedef struct VmafModel VmafModel;
 
-int vmaf_model_load_from_path(VmafModel **model, const char *path);
+int vmaf_model_load_from_path(VmafModel **model, VmafModelConfig *config);
```

Comment:
`cfg` instead of `config`, not really a big problem but this is how `VmafConfiguration`s are referred to, I think.